modules/registry-securing-manually.adoc

```diff
@@ -16,30 +16,30 @@ Manually secure the registry to serve traffic via TLS:
 example:
```

Comment:
This section is currently commented out of the documentation, but I made some requested changes just in case.

Reply:
I'd say we shouldn't recommend to secure the registry manually anymore. Inside the cluster it is secured by default. Outside of the cluster it can be accessed only through routes, which should have documentation about certificates and all this stuff.

Reply:
Thanks, @dmage! It sounds like we might want to move the entire Securing and exposing the registry topic: https://docs.openshift.com/container-platform/4.1/registry/securing-exposing-registry.html

Is that correct?

Reply:
@bmcelvee yes, the whole section of registry-securing-manually.adoc should be removed.

Reply:
Thanks! I'll go ahead and remove the entire assembly.